.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+    "python.pythonPath": "env/bin/python",
+    "python.linting.pylintEnabled": true
+}
```

Comment:
Please add a new line to the end of this file.

Reply:
Done; thanks!